package.json

```diff
@@ -14,27 +14,26 @@
   },
   "scripts": {
     "postinstall": "composer install",
-    "dev": "webpack --watch --mode=development --output=js/dist/[name].js",
-    "build": "webpack --mode=production --output=js/dist/[name].js",
+    "dev": "wp-scripts start",
+    "build": "wp-scripts build",
     "lint": "npm-run-all --parallel lint:*",
     "lint:js": "eslint js/src",
     "lint:php": "composer lint",
     "test": "npm-run-all --parallel test:*",
     "test:php": "composer test",
+    "test:php:no-coverage": "composer test -- --no-coverage",
     "coverage": "npm-run-all --parallel coverage:*",
     "coverage:php": "composer coverage"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@wordpress/babel-preset-default": "4.4.0",
```

Comment:
This is also [bundled with `wp-scripts`](https://github.com/WordPress/gutenberg/blob/f198997e2c8e377423beb230ce5283914076d396/packages/scripts/package.json#L35) so we can probably remove it.

Reply:
Good idea, this is applied in https://github.com/xwp/block-extend/pull/10/commits/b7a7288d1d0114f08031c0f3c9f86ab52a4a76d4